tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -352,17 +394,24 @@ describe(__filename, () => {
       <a href="https://support.mozilla.org/kb/change-primary-email-address-firefox-accounts">Need help?</a></p>`
     );
 
+    expect(root.find('.UserProfileEdit-homepage--help')).toHaveLength(0);
+    expect(root.find('.UserProfileEdit-biography--help')).toHaveLength(0);
+    expect(root.find('.UserProfileEdit-notifications--help')).toHaveLength(0);
```

Comment:
hmm to me this maybe could be a little clearer.. i mean this help text should be displayed when user is an dev or artist - should that be made cleared by test description? maybe something like ~ `renders help text section for email only if user is not a developer/artist` ?